daemon/migrations/20220502172229_create_time_to_first_position_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS time_to_first_position (
+    id integer PRIMARY KEY autoincrement,
+    taker_id text NOT NULL,
```

Comment:
Imho this should be `UNIQUE`

Reply:
There is a unique constraint further down!

Reply:
Is there a specific reason / advantage for (only) creating the `UNIQUE` constraint implicitly through adding an index? 
I agree with @bonomat that it would be easier to understand if we'd have (an additional) `UNIQUE` constraint here.

Reply:
I probably just forgot the `UNIQUE` label!